public/app/plugins/datasource/elasticsearch/components/QueryEditor/BucketAggregationsEditor/SettingsEditor/FiltersSettingsEditor/state/actions.ts

```diff
@@ -1,16 +1,6 @@
+import { createAction } from '@reduxjs/toolkit';
 import { Filter } from '../../../aggregations';
-import { FilterAction, ADD_FILTER, REMOVE_FILTER, CHANGE_FILTER } from './types';
 
-export const addFilter = (): FilterAction => ({
-  type: ADD_FILTER,
-});
-
-export const removeFilter = (index: number): FilterAction => ({
-  type: REMOVE_FILTER,
-  payload: { index },
-});
-
-export const changeFilter = (index: number, filter: Filter): FilterAction => ({
-  type: CHANGE_FILTER,
-  payload: { index, filter },
-});
+export const addFilter = createAction('@bucketAggregations/filter/add');
```

Comment:
Do we have any guidelines on naming conventions for action names? I used slightly different way in Graphite, but would be nice to have it consistent.

Reply:
i don't think so, but it would be extremely nice